src/Blocks/custom/form/components/form-buckaroo-options.js

```diff
@@ -1,146 +1,132 @@
 import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 import { SelectControl, TextControl, TextareaControl, BaseControl, ToggleControl } from '@wordpress/components';
-
+import { getAttrKey } from '@eightshift/frontend-libs/scripts/helpers';
+import manifest from '../manifest.json';
 
 export const FormBuckarooOptions = (props) => {
-  const {
-    blockClass,
-    service,
-    paymentDescription,
-    emandateDescription,
-    sequenceType,
-    isSequenceTypeOnFrontend,
-    redirectUrl,
-    redirectUrlError,
-    redirectUrlReject,
-    onChangeIsSequenceTypeOnFrontend,
-    onChangeService,
-    onChangePaymentDescription,
-    onChangeEmandateDescription,
-    onChangeSequenceType,
-    onChangeRedirectUrl,
-    onChangeRedirectUrlError,
-    onChangeRedirectUrlReject,
-  } = props;
-
-  const MAX_CHARS_IN_EMANDATE_DESCRIPTION_FIELD = 70;
+	const {
+		attributes,
+		setAttributes,
+		blockClass,
+		formBuckarooService,
```

Comment:
It's in a block, it won't (and can't) be used anywhere else. Also not in here because this technically isn't a lvl down in the component tree. (i.e. `props` will unnecessarily add a prefix here) These options are just here so the `form-options.js` isn't huge.